WeatherApp/Views/WeatherTableViewCell.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  TableViewCell.swift
+//  WeatherApp
+//
+//  Created by Lucija Balja on 03/08/2020.
+//  Copyright © 2020 Lucija Balja. All rights reserved.
+//
+
+import UIKit
+
+class WeatherTableViewCell: UITableViewCell {
+    let containerView = UIView()
+    let weatherIcon = UIImageView()
+    let cityLabel = UILabel()
+    let currentTempLabel = UILabel()
+    let minTempLabel = UILabel()
+    let maxTempLabel = UILabel()
+    
+    var weather: WeatherData? {
+        didSet {
+            guard let weatherData = weather else { return }
```

Comment:
Move all of the code to a func, for example  `func updateUI(for weatherData: WeatherData)`